<ui version="4.0" >
 <class>ShaderConfigDlgClass</class>
 <widget class="QWidget" name="ShaderConfigDlgClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ShaderConfigDlg</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <property name="leftMargin" >
      <number>9</number>
     </property>
     <property name="topMargin" >
      <number>9</number>
     </property>
     <property name="rightMargin" >
      <number>9</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" native="1" name="widget" >
         <property name="minimumSize" >
          <size>
           <width>260</width>
           <height>80</height>
          </size>
         </property>
         <widget class="QCheckBox" name="toMulti" >
          <property name="geometry" >
           <rect>
            <x>178</x>
            <y>30</y>
            <width>81</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text" >
           <string>Multisample</string>
          </property>
         </widget>
         <widget class="QComboBox" name="whatCombo" >
          <property name="geometry" >
           <rect>
            <x>58</x>
            <y>2</y>
            <width>91</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_5" >
          <property name="geometry" >
           <rect>
            <x>156</x>
            <y>5</y>
            <width>21</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font" >
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="text" >
           <string>To</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="incPoly" >
          <property name="geometry" >
           <rect>
            <x>13</x>
            <y>40</y>
            <width>131</width>
            <height>18</height>
           </rect>
          </property>
          <property name="toolTip" >
           <string>Draw the polygons of the scene with shaders on</string>
          </property>
          <property name="text" >
           <string>Polygons in program</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="incPoints" >
          <property name="geometry" >
           <rect>
            <x>13</x>
            <y>60</y>
            <width>131</width>
            <height>18</height>
           </rect>
          </property>
          <property name="toolTip" >
           <string>Draw the points of the scene with shaders on</string>
          </property>
          <property name="text" >
           <string>Points in program</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4" >
          <property name="geometry" >
           <rect>
            <x>12</x>
            <y>5</y>
            <width>51</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font" >
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="text" >
           <string>Render</string>
          </property>
         </widget>
         <widget class="QComboBox" name="toCombo" >
          <property name="geometry" >
           <rect>
            <x>178</x>
            <y>3</y>
            <width>69</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="ExGroupBox" name="pointGrp" >
         <property name="minimumSize" >
          <size>
           <width>260</width>
           <height>90</height>
          </size>
         </property>
         <property name="title" >
          <string>Points</string>
         </property>
         <widget class="QCheckBox" name="vertexPointSize" >
          <property name="geometry" >
           <rect>
            <x>10</x>
            <y>20</y>
            <width>211</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text" >
           <string>GL_VERTEX_PROGRAM_POINT_SIZE</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="spiriteReplace" >
          <property name="geometry" >
           <rect>
            <x>20</x>
            <y>64</y>
            <width>191</width>
            <height>18</height>
           </rect>
          </property>
          <property name="toolTip" >
           <string>Center a texture around a point</string>
          </property>
          <property name="text" >
           <string>Point spirite coordinates replace</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="pointSpirites" >
          <property name="geometry" >
           <rect>
            <x>10</x>
            <y>42</y>
            <width>191</width>
            <height>18</height>
           </rect>
          </property>
          <property name="toolTip" >
           <string>Center a texture around a point</string>
          </property>
          <property name="text" >
           <string>Point spirites</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="ExGroupBox" name="geomGrp" >
         <property name="minimumSize" >
          <size>
           <width>330</width>
           <height>80</height>
          </size>
         </property>
         <property name="title" >
          <string>Geometry Shader (need compile)</string>
         </property>
         <widget class="QLabel" name="label" >
          <property name="geometry" >
           <rect>
            <x>10</x>
            <y>21</y>
            <width>70</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text" >
           <string>Input type:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2" >
          <property name="geometry" >
           <rect>
            <x>10</x>
            <y>49</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text" >
           <string>Output type:</string>
          </property>
         </widget>
         <widget class="QComboBox" name="geomInput" >
          <property name="geometry" >
           <rect>
            <x>80</x>
            <y>18</y>
            <width>111</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QComboBox" name="geomOutput" >
          <property name="geometry" >
           <rect>
            <x>80</x>
            <y>47</y>
            <width>111</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_3" >
          <property name="geometry" >
           <rect>
            <x>202</x>
            <y>49</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text" >
           <string>Vertices out:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="geomVtxCount" >
          <property name="geometry" >
           <rect>
            <x>268</x>
            <y>47</y>
            <width>51</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>0</width>
       <height>999</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PParamsWidget" native="1" name="paramContainer" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>PParamsWidget</class>
   <extends>QWidget</extends>
   <header>PParamsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ExGroupBox</class>
   <extends>QGroupBox</extends>
   <header>MyLib\ExGroupBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
